cmd/list.go

```diff
@@ -21,8 +42,45 @@ var listCmd = &cobra.Command{
 	RunE:       list,
 }
 
+type fmtFn func(writer io.Writer, names []string) error
+
+func fmtPlain(writer io.Writer, names []string) error {
+	for _, name := range names {
+		_, err := fmt.Fprintf(writer, "%s\n", name)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func fmtJSON(writer io.Writer, names []string) error {
+	encoder := json.NewEncoder(writer)
+	return encoder.Encode(names)
+}
+
+func fmtYAML(writer io.Writer, names []string) error {
+	encoder := yaml.NewEncoder(writer)
+	return encoder.Encode(names)
+}
+
+var formats = map[string]fmtFn{
+	"plain": fmtPlain,
+	"json":  fmtJSON,
+	"yaml":  fmtYAML,
+}
```

Comment:
Nitpick, but can we move this up top?  It threw me for a loop for a minute wondering where `formats` came from.